doc/modules/ROOT/pages/graph-object.adoc

```diff
@@ -145,7 +145,7 @@ assert G.node_count() == result["nodeCount"]
 [[construct]]
 == Constructing a graph from DataFrames
 
-Instead of projecting a graph from the Neo4j database it is also possible to construct new graphs using pandas `DataFrames` from the client.
+In addition to projecting a graph from the Neo4j database, it is also possible to create in-memory graphs directly from pandas `DataFrame` objects.
```

Comment:
I think it was decided by someone at some point that we should refer to GDS graphs as "projections" rather than the less abstract "in-memory graphs". Personally I don't have an opinion on the subject

Reply:
I think in this case I can actually omit the "in-memory" part and just leave it as "to create graphs", if that's ok. 

More in general I thought a "projection" should be the operation of creating a graph rather than the graph itself, but I'm happy to discuss.